manager/api/misc_api.py

```diff
@@ -281,6 +281,10 @@ def get(self, id1):
         """
```

Comment:
Why all these changes now? They seem fine, but more like they should have been committed a long time ago?

Reply:
Yeah, seems like I was sitting on them since the hackathon.